libs/sdk-embedding/api/sdk-embedding.api.md

```diff
@@ -340,6 +360,11 @@ export namespace EmbeddedGdc {
     export type RelativeType = "relative";
     // (undocumented)
     export type RemoveFilterItem = IRemoveDateFilterItem | IRemoveAttributeFilterItem | IRemoveRankingFilterItem;
+    // (undocumented)
+    export type ResolvableFilter = Model.IDateFilter | Model.IAttributeFilter;
+    // (undocumented)
+    export type ResolvedDateFilterValues = IResolvedDateFilterValue[];
+        {};
```

Comment:
This `{};` was generated automatically, right? I wonder why..

Reply:
fixed